### DISCUSSION TITLE
Error when installing WSL: 0x80080005 Server execution failed

### DISCUSSION THREAD

**Discussion by samuelkkitson** [Cyfrin/foundry-full-course-cu#2831] [Q&A]
I have used the wsl --install command then rebooted the device and openend up Ubuntu but then I get this error message:

WslRegisterDistribution failed with error: 0x80080005 Error: 0x80080005 Server execution failed

I have already looked up some solutions, I tried switching off the Windows Distribution for Linux environment somewhere in the settings, reboot and the back on and reboot.

While Rebooting I noticed that it tries to Update Windows but everytime it says after a while: Failure trying to Configure Windows Features (or sometimes Update): Reverting Features (or Changes). 

I am on Windows 10. My Laptop is not eligble for Windows 11. Please help

**Comment by EngrPips** [comment]
Hello @samuelkkitson, Honestly, I don't use Windows, and I don't have any experience as regards `wsl` but I trust someone who knows will see this and help as soon as possible.

**Reply by samuelkkitson** [reply]
Yes hopefully. Do you might know if the wsl installation is really necessary to continue the course or is it possible to continue without it?

**Reply by EngrPips** [reply]
If you are using a Windows system, then you need it. I hear people install Ubuntu, too, but I am unsure what that is. Maybe you can look into that. I have been using Mac my whole life, so pardon my knowledge regarding this.
